config/subsidy-application-management/forms/contact-tracing/versions/20210617184000-summer-update/form.json

```diff
@@ -0,0 +1,96 @@
+{
+  "source": {
+    "prefixes": [
+      "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>",
+      "PREFIX dct: <http://purl.org/dc/terms/>",
+      "PREFIX lblodSubsidie: <http://lblod.data.gift/vocabularies/subsidie/>",
+      "PREFIX schema: <http://schema.org/>",
+      "PREFIX foaf: <http://xmlns.com/foaf/0.1/>",
+      "PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>"
+    ],
+    "properties": [
+      "lblodSubsidie:isCollaboration",
+      "lblodSubsidie:collaborator",
+      "lblodSubsidie:extraContactTracingMeasuresApril",
```

Comment:
Didn't Felix mention something about making these predicates also mention the year, if we did not clean them up?

Reply:
I think the end decision was no mention of the year in the predicate as we can get back the period of time from the Reeks itself